docs/azure/sdk/resource-management.md

```diff
@@ -132,19 +158,19 @@ Imagine that our company requires all virtual machines to be tagged with the own
 var armClient = new ArmClient(new DefaultAzureCredential());
 
 // Next we get a resource group object
-// ResourceGroup is a [Resource] object from above
-Subscription subscription = await armClient.GetDefaultSubscriptionAsync();
-ResourceGroup resourceGroup = await subscription.GetResourceGroups().GetAsync("myRgName");
+// ResourceGroup is a [ResourceName] object from above
+SubscriptionResource subscription = await armClient.GetDefaultSubscriptionAsync();
+ResourceGroupResource resourceGroup = (await subscription.GetResourceGroups().GetAsync("myRgName").Value);
```

Comment:
The Value might be redundant and is the parenthesis matching with each other? It looks not correct